<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>NewCustomerDialog</class>
 <widget class="QDialog" name="NewCustomerDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>New customer</string>
  </property>
  <widget class="QWidget" name="formLayoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>381</width>
     <height>407</height>
    </rect>
   </property>
   <layout class="QFormLayout" name="formLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="lastNameLabel">
      <property name="text">
       <string>Last name :</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLineEdit" name="lastNameLineEdit"/>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="firstNameLabel">
      <property name="text">
       <string>First name :</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QLineEdit" name="firstNameLineEdit"/>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="addressLabel">
      <property name="text">
       <string>Address :</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QLineEdit" name="addressLineEdit"/>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="cityLabel">
      <property name="text">
       <string>City :</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QLineEdit" name="cityLineEdit"/>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="postalCodeLabel">
      <property name="text">
       <string>Postal code :</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QLineEdit" name="postalCodeLineEdit"/>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="dayOfConsultationLabel">
      <property name="text">
       <string>Day of consultation :</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QDateEdit" name="dayOfConsultationDateEdit">
      <property name="calendarPopup">
       <bool>true</bool>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="durationLabel">
      <property name="text">
       <string>Duration :</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QTimeEdit" name="durationTimeEdit">
      <property name="calendarPopup">
       <bool>false</bool>
      </property>
      <property name="time">
       <time>
        <hour>1</hour>
        <minute>0</minute>
        <second>0</second>
       </time>
      </property>
     </widget>
    </item>
    <item row="7" column="0">
     <widget class="QLabel" name="priorityLabel">
      <property name="text">
       <string>Priority :</string>
      </property>
     </widget>
    </item>
    <item row="7" column="1">
     <widget class="QComboBox" name="priorityComboBox"/>
    </item>
    <item row="8" column="0">
     <widget class="QLabel" name="ressourcesLabel">
      <property name="text">
       <string>Ressources :</string>
      </property>
     </widget>
    </item>
    <item row="8" column="1">
     <widget class="QListWidget" name="resourcesListWidget">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>73</height>
       </size>
      </property>
      <property name="selectionMode">
       <enum>QAbstractItemView::ExtendedSelection</enum>
      </property>
      <property name="isWrapping" stdset="0">
       <bool>false</bool>
      </property>
      <property name="resizeMode">
       <enum>QListView::Fixed</enum>
      </property>
     </widget>
    </item>
    <item row="9" column="0">
     <widget class="QLabel" name="commentLabel">
      <property name="text">
       <string>Comment :</string>
      </property>
     </widget>
    </item>
    <item row="9" column="1">
     <widget class="QLineEdit" name="commentLineEdit"/>
    </item>
    <item row="10" column="0">
     <widget class="QLabel" name="phoneLabel">
      <property name="text">
       <string>Phone :</string>
      </property>
     </widget>
    </item>
    <item row="10" column="1">
     <widget class="QLineEdit" name="phoneLineEdit"/>
    </item>
    <item row="11" column="0">
     <widget class="QPushButton" name="cancelButton">
      <property name="text">
       <string>Cancel</string>
      </property>
      <property name="default">
       <bool>false</bool>
      </property>
     </widget>
    </item>
    <item row="11" column="1">
     <widget class="QPushButton" name="okButton">
      <property name="text">
       <string>OK</string>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
